Blockzilla/BrowserViewController.swift

```diff
@@ -980,12 +977,15 @@ class BrowserViewController: UIViewController {
 
             let shareMenu = UIMenu(options: .displayInline, children: shareItems.compactMap { $0 })
             actions.append(shareMenu)
+            actions.append(UIMenu(options: .displayInline, children: [UIAction(whatsNewItem), UIAction(settingsItem)]))
 
         } else {
-            actions.append(UIMenu(options: .displayInline, children: [UIAction(helpItem)]))
+            var actionItems: [UIMenuElement] = [UIAction(whatsNewItem)]
+            actionItems.append(UIAction(helpItem))
+            actionItems.append(UIAction(settingsItem))
+            let actionMenu = UIMenu(options: .displayInline, children: actionItems)
```

Comment:
Minor: Can inlining work?
Something like `let actionMenu = UIMenu(options: .displayInline, children: [UIAction(whatsNewItem), UIAction(helpItem), UIAction(settingsItem)])`?

Reply:
yes, I think it will work since I used it for the case with two options for settings and whats new. I will add this change. Thank you!